packages/chess-contracts/src/chess-contract.ts

```diff
@@ -211,10 +211,12 @@ export class ChessContractHelper {
     const network = this.computer.getNetwork()
     const n = networks.getNetwork(chain, network)
     const addy = address.fromPublicKey(this.computer.db.wallet.publicKey, 'p2pkh', n)
-    const utxos = await this.computer.db.wallet.restClient.getFormattedUtxos(addy)
+    const utxos = await this.computer.getUTXOs({ address: addy, verbosity: 1 })
     let paid = 0n
     while (paid < Number(this.satoshis) / 2 && utxos.length > 0) {
-      const { txId, vout, satoshis } = utxos.pop()!
+      const { rev, satoshis } = utxos.pop()!
```

Comment:
We should consider using parseRev here. If it is not exported we should export it